
Turn 1
================================================================
error: communication error with bot: Problem writing to pipe, errno was: 0 (No error)
Last input received was:

  File "MyBot.py", line 5, in <module>
    import hlt
  File "C:\Users\Matt\Documents\saltpicus\hlt\__init__.py", line 3, in <module>
    from . import commands, entity, game_map, networking, constants
  File "C:\Users\Matt\Documents\saltpicus\hlt\entity.py", line 8, in <module>
    class Entity(abc.ABC):
NameError: name 'abc' is not defined

